pontoon/base/static/css/boilerplate.css

```diff
@@ -1,7 +1,7 @@
 /*
   HTML5 ✰ Boilerplate
 
-  style.css contains a reset, font normalization and some base styles.
+  style.css contains font normalization and some base styles.
```

Comment:
Nit: that comment is still wrong - it talks about `style.css`, but this is `boilerplate.css`.

Reply:
Right, and my change was dumb: this file actually contains a CSS reset…